lesson02/assignment/css/style.scss

```diff
@@ -1,6 +1,5 @@
 // Import partials here
-@import 'base.normalize';
+// @import 'base.normalize';
```

Comment:
I think you'll want to use this partial. CSS normalization is an excellent way to ensure all browsers behave similarly.